R/src.translate.env.src.presto.R

```diff
@@ -14,6 +18,9 @@ src_translate_env.src_presto <- function(x) {
         sql_type <- toupper(dbDataType(Presto(), type))
         dplyr::build_sql('CAST(', column, ' AS ', dplyr::ident(sql_type), ')')
       },
+      `[` = .subscript,
+      `[[` = .subscript,
+      length = dplyr::sql_prefix("cardinality"),
```

Comment:
Oops, this slipped in by mistake. But now that we have a discussion, my plan was to add `length` -> `cardinality`, and `nchar` -> `length`. After all, our goal is to translate R semantics to presto land.